### ISSUE TITLE
Broken Usage Example in README

### ISSUE THREAD

**Issue by bsingr** [agektmr/binarize.js#7]
Hey folks,

the current README gives the following example:

```
var buffer = binarize.pack(function(object) {  
   ...  
});
```

However, the pack function expects an object as the first argument and the callback function as the second argument. Also it won’t return a buffer, but invoke the callback function with a DataView.

For me, the following works:

```
var object;
object = {
  name: 'Eiji Kitamura',
  array: [1, 2, 3],
  object: {
    name: 'Eiji Kitamura',
    hello: 'こんにちは'
  }
};
binarize.pack(object, function(dataView) {
  console.log('packed!', dataView);
  binarize.unpack(dataView.buffer, function(unpacked) {
    console.log('unpacked', unpacked);
  });
});
```

What is the README describing? Is it just outdated?

Regards,
—dpree

**Comment by agektmr** [comment]
You are absolutely correct. [AudioStreamer](https://github.com/agektmr/AudioStreamer/blob/master/public/js/AudioStreamer.js#L148) has a real world example. I will fix the documentation.
Thanks for reporting!
